app/src/components/common/icons/IconBack.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const IconBack = () => (
+  <svg fill="none" height="12" viewBox="0 0 18 12" width="18" xmlns="http://www.w3.org/2000/svg">
+    <path d="M18 5H3.83L7.41 1.41L6 0L0 6L6 12L7.41 10.59L3.83 7H18V5Z" />
+  </svg>
+)
```

Comment:
I don't think its necessary to import another Icon here I used on my proposal details pr IconArrowBack with slight modification to support this color https://github.com/protofire/omen-exchange/pull/2208/files#

Reply:
Cheers dude, copied!